client/src/lib/components/map/tile.svelte

```diff
@@ -109,7 +110,7 @@
                background-size: cover;
                background-position: center;`}
 >
-  {#if isOwner}
+  {#if isOwner() && scale > 1.5}
```

Comment:
isOwner should not be called (or at least i'm getting an error from my side if u can check)
```suggestion
  {#if isOwner && scale > 1.5}
```

Reply:
Good catch, this is a leftover from a bad merge